Entity/Notification.php

```diff
@@ -84,6 +93,14 @@ public function getBody()
     }
 
     /**
+     * @return File[]
+     */
+    public function getAttachments()
+    {
+        return $this->attachments;
+    }
```

Comment:
If this is an array collection and if you are expecting to return the list of attachments, think about calling `toArray`